src/compute-shortname.js

```diff
@@ -179,6 +179,7 @@ function completeWithSeriesAndLevel(shortname, url, forkOf) {
   // digits, as in "iso18181-2".
   if (seriesBasename.match(/^ecma-/) ||
       seriesBasename.startsWith("iso") ||
+      seriesBasename.endsWith("base64") ||
```

Comment:
Hmm, I was thinking of a more generic "no series version for TC39 proposals" exception, since in practice these proposals do not, and should not, contain any version number.

Looking into it, the exception could actually be "no series version for TC39 specs" since, at least for now, we don't try to keep track of versions of TC39 specs:

```suggestion
      url.match(/^https:\/\/tc39\.es\//) ||
```

This also allows to drop the first `seriesBasename.match(/^ecma-/)` exception as underlying URLs also start with `https://tc39.es/`.

Reply:
much more elegant, indeed :) will fix

Maybe the generalization of this is using an allowlist for organizations that do use series version? (or a blocklist of the reverse); that's probably not practical at compute-shortname stage, but could be done in a clean up phase?

Reply:
Ah well, the approach based on the URL is not going to work as-is, due to the fact that shortname was made explicit, which effectively replaces the URL in the call to `computeShortname`. See #1219.

I'll update to a check on the shortname instead.